SparkPost/2web2ui
# AC-1932 - Track all pageviews

AC-1932 - Track all pageviews 

### What Changed

- Added tracking for page redirects and also added supporting cypress tests for the same
- Added tracking for url changes and also added supporting cypress tests for the same

### How To Test

- For checking redirects 
    - Enter /reports/summary, you should be redirected to signals/analytics
        - Expectation: reports/summary should be tracked in segment along with signal/analytics
- For checking url changes
    - Best places to test this is domains and signal analytics page, try changing filters on these pages.
        - Expectation: all query param changes should be tracked in segment
------------------------------------------------------------------------------------------------
- For testing the utm change, we can use the url below
   - /templates?utm_campaign=fake
### To Do
